pandas/tests/reshape/test_pivot.py

```diff
@@ -353,6 +353,29 @@ def test_pivot_periods(self):
         pv = df.pivot(index='p1', columns='p2', values='data1')
         tm.assert_frame_equal(pv, expected)
 
+    def test_pivot_with_multi_values(self):
+        df = pd.DataFrame({'foo': ['one', 'one', 'one', 'two', 'two', 'two'],
+                           'bar': ['A', 'B', 'C', 'A', 'B', 'C'],
+                           'baz': [1, 2, 3, 4, 5, 6],
+                           'zoo': ['x', 'y', 'z', 'q', 'w', 't']})
+
+        results = df.pivot(index='zoo', columns='foo', values=['bar', 'baz'])
+
+        data = [[None, 'A', None, 4],
+                [None, 'C', None, 6],
+                [None, 'B', None, 5],
+                ['A', None, 1, None],
+                ['B', None, 2, None],
```

Comment:
parmaterize this with values a list, tuple, np.array, pd.Series, pd.Index (should all act the same)